### PR TITLE
7629 Use more efficient version of findFiles()

### PR DESCRIPTION
**Pull request by APriestman**
Depends on https://github.com/sleuthkit/sleuthkit/pull/2446

### CODE REVIEW

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/DataSourceUsageAnalyzer.java [hunk 1]
```diff
@@ -163,8 +163,9 @@ private void createDataSourceUsageArtifact(String dataSourceUsageDescription) th
     private void checkIfOsSpecificVolume(ExtractOs.OS_TYPE osType) throws TskCoreException {
         FileManager fileManager = currentCase.getServices().getFileManager();
         for (String filePath : osType.getFilePaths()) {
-            for (AbstractFile file : fileManager.findFiles(dataSource, FilenameUtils.getName(filePath), FilenameUtils.getPath(filePath))) {
-                if ((file.getParentPath() + file.getName()).equals(filePath)) {
+            for (AbstractFile file : currentCase.getSleuthkitCase().getFileManager().findFilesExactNameExactPath(dataSource, 
+                    FilenameUtils.getName(filePath), FilenameUtils.getPath(filePath))) {
+               if ((file.getParentPath() + file.getName()).equals(filePath)) {
```
Comment (bcarrier) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/DataSourceUsageAnalyzer.java:168]: I think we can get rid of this now that we know it's exact match.

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractOs.java [hunk 2]
```diff
@@ -117,9 +117,9 @@ private void checkForOSFiles(OS_TYPE osType) throws TskCoreException {
      *         search for
      */
     private AbstractFile getFirstFileFound(List<String> pathsToSearchFor) throws TskCoreException{
-        FileManager fileManager = currentCase.getServices().getFileManager();
         for (String filePath : pathsToSearchFor) {
-            for (AbstractFile file : fileManager.findFiles(dataSource, FilenameUtils.getName(filePath), FilenameUtils.getPath(filePath))) {
+            
+            for (AbstractFile file : currentCase.getSleuthkitCase().getFileManager().findFilesExactNameExactPath(dataSource, FilenameUtils.getName(filePath), FilenameUtils.getPath(filePath))) {
                 if ((file.getParentPath() + file.getName()).equals(filePath)) {
```
Comment (bcarrier) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractOs.java:123]: Same.

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java [hunk 3]
```diff
@@ -1684,12 +1684,11 @@ private void parseOfficeTrustRecords(AbstractFile regFile, BufferedReader reader
...
-        org.sleuthkit.autopsy.casemodule.services.FileManager fileManager = currentCase.getServices().getFileManager();
         String fileName = FilenameUtils.getName(filePathName);
         String filePath = FilenameUtils.getPath(filePathName);
         List<AbstractFile> sourceFiles;
         try {
-            sourceFiles = fileManager.findFiles(dataSource, fileName, filePath); //NON-NLS
+            sourceFiles = currentCase.getSleuthkitCase().getFileManager().findFilesExactNameExactPath(dataSource, fileName, filePath);
             if (!sourceFiles.isEmpty()) {
                 for (AbstractFile sourceFile : sourceFiles) {
                     if (sourceFile.getParentPath().endsWith(filePath)) {
```
Comment (bcarrier) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:1694]: I can't tell if this is here to get exact matches or not...

Comment (APriestman) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:1694]: I'm pretty sure it is. I'll remove it.

**Review by eugene7646 — APPROVED** [review]

**Pull request merged**